src/databases/redis.js

```diff
@@ -21,7 +21,7 @@ export function saveParsedEvent(event: Event): Promise<any> {
                 return reject(err);
```

Comment:
This looks good overall. The only thing that can be improved that I notice is: is `reject(err)` going to return a `Promise<boolean>`?
Also, `setex()` is not typed, so flow can not infer the correct return type, therefore not throwing an error, but I guess that typing `setex()` is not a priority for now. (Also, maybe we can check flow-typed for redis types)